MAVProxy/modules/mavproxy_paramedit/param_editor.py

```diff
@@ -155,6 +156,12 @@ def process_mavlink_packet(self, m):
                 del self.paramchanged[m.param_id.upper()]
             self.gui_event_queue.put(ParamEditorEvent(
                 ph_event.PEGE_WRITE_SUCC, paramid=m.param_id.upper(), paramvalue=m.param_value))
+        if mtype in ['RC_CHANNELS', 'RC_CHANNELS_RAW']:
+            rc_received = float(getattr(m, 'chan%u_raw' % self.mpstate.module('param').mav_param['FLTMODE_CH']))
```

Comment:
we need to range check FLTMODE_CH. The user can set it to zero (or any invalid channel) to disable changing modes using RC. Also, the name of the parameter varies with firmware, on rover it is MODE_CH and the modes are MODE1 to MODE6, so we'll need an abstraction for that

Reply:
Done